Frontend/Pages/Projects/BenefitsAndRisks/EqualitiesImpactAssessment.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@page "/project/{urn}/benefits/equalities-impact-assessment"
+@using Microsoft.AspNetCore.Mvc.ModelBinding
+@model Frontend.Pages.Projects.BenefitsAndRisks.EqualitiesImpactAssessment
+
+@{
+    ViewBag.Title = (!ViewData.ModelState.IsValid ? "Error: " : "") + "Has an Equalities Impact Assessment been considered?";
+    Layout = "_Layout";
+    var formClasses = ViewData.ModelState.GetFieldValidationState($"{nameof(Model.EqualitiesImpactAssessmentViewModel)}.{nameof(Model.EqualitiesImpactAssessmentViewModel.EqualitiesImpactAssessmentConsidered)}") == ModelValidationState.Invalid ? "govuk-form-group--error" : "";
```

Comment:
Small thing but in my view this line is too long which makes it difficult to read, consider putting a new line in here or breaking out some of the parameters into variables.